frontend/packages/core/src/AppLayout/user.tsx

```diff
@@ -1,37 +1,79 @@
 import React from "react";
+import styled from "@emotion/styled";
 import {
   Avatar as MuiAvatar,
+  Box,
   ClickAwayListener,
-  Grid,
-  Grow,
+  Grow as MuiGrow,
   IconButton,
+  ListItemIcon,
+  ListItemText,
+  MenuItem as MuiMenuItem,
   MenuList,
-  Paper,
+  Paper as MuiPaper,
   Popper,
   Typography,
 } from "@material-ui/core";
 import Cookies from "js-cookie";
 import jwtDecode from "jwt-decode";
-import styled from "styled-components";
-
-const UserPhoto = styled(IconButton)`
-  ${({ theme }) => `
-  padding: 1px 0 ${theme.spacing(0.5)}px 12px;
-  margin-right: ${theme.spacing(0.5)}px;
-  `}
-`;
-
-const Avatar = styled(MuiAvatar)`
-  ${({ theme }) => `
-  background-color: ${theme.palette.text.secondary}
-  `}
-`;
-
-const Initials = styled(Typography)`
-  ${({ theme }) => `
-  color: ${theme.palette.accent.main}
-  `}
-`;
+
+const UserPhoto = styled(IconButton)({
+  padding: "12px",
+  "&:hover": {
+    background: "#2d3db4",
+  },
+  "&:active": {
+    background: "#2938a5",
+  },
+});
+
+const Avatar = styled(MuiAvatar)({
+  backgroundColor: "#dce7f4",
+  height: "28px",
+  width: "28px",
+});
+
+const AvatarBackdrop = styled(MuiAvatar)({
+  backgroundColor: "#f6faff",
+  height: "32px",
+  width: "32px",
```

Comment:
```suggestion
  border: "2px solid #f6faff",
  borderRadius: "50%",
```

Reply:
oh nice! i tried a variation of this but couldn't get it right. thank you, updated